guides/common/modules/proc_upgrading-a-disconnected-project-server.adoc

```diff
@@ -44,107 +44,68 @@ This prevents the upgrade process from overwriting your DNS and DHCP configurati
-----
-
-... Add the `baseurl` directive:
-+
-[options="nowrap"]
-----
-baseurl=file:///media/sat6/Satellite
-----
-
-. Configure the {ProjectName} Maintenance repository from the ISO file.
-
-.. Copy the ISO file's repository data file for {ProjectName} Maintenance packages:
-+
-[options="nowrap"]
-----
-# cp /media/sat6/Maintenance/media.repo /etc/yum.repos.d/satellite-maintenance.repo
-----
-
-.. Edit the `/etc/yum.repos.d/satellite-maintenance.repo` file:
-+
-[options="nowrap"]
-----
-# vi /etc/yum.repos.d/satellite-maintenance.repo
-----
-
-... Change the default `InstallMedia` repository name to `{Project}-Maintenance`:
+* {RepoRHEL9BaseOS}
+* {RepoRHEL9AppStream}
+* {RepoRHEL9ServerSatelliteServerProjectVersion}
+.. Export the repositories in a syncable format:
 +
-[options="nowrap" subs="+quotes,attributes"]
-----
-[{Project}-Maintenance]
-----
-
-.. Add the `baseurl` directive:
-+
-[options="nowrap"]
+[options="nowrap" subs="+quotes,verbatim,attributes"]
 ----
-baseurl=file:///media/sat6/Maintenance/
+$ hammer content-export complete repository \
+--id=_Repo_ID_ --format=syncable
 ----
+.. Copy the exported directories to the disconnected {ProjectServer}.
 
-ifdef::katello,satellite,orcharhino[]
-. If you use {advisorengine} in your environment, update to the latest images:
-* To update using the {Project} installation ISO image, set up containers on your {ProjectServer}:
+. On the connected server, export the required repositories in a syncable format and copy the exported directories to the disconnected server.
+. On the disconnected server, stop all {Project} services:
 +
-[options="nowrap", subs="+quotes,verbatim,attributes"]
-----
-# /media/sat6/containers/setup_containers
-----
-* Alternatively, to update by transferring the latest image from a connected system to a disconnected {ProjectServer}, follow these steps:
-... On the connected system, export the container image:
-+
-[options="nowrap", subs="+quotes,verbatim,attributes"]
+[options="nowrap" subs="+quotes,verbatim,attributes"]
 ----
-# skopeo copy containers-storage:registry.redhat.io/satellite/iop-advisor-engine-rhel9:{ProjectVersion} \
-oci-archive:/tmp/iop-advisor-engine.tar
+# {foreman-maintain} service stop
 ----
-... Transfer the `iop-advisor-engine.tar` file to the disconnected {ProjectServer}.
-... On the disconnected {ProjectServer}, import the container image:
+. On the disconnected server, enable access to the exported content by configuring a repository:
+.. Identify the location of the copied exported directories from the syncable export on the connected server.
+.. Create the `/etc/yum.repos.d/upgrade.repo` file and update the `baseurl` directive to point to that location:
 +
-[options="nowrap", subs="+quotes,verbatim,attributes"]
+[options="nowrap" subs="attributes"]
 ----
-# skopeo copy oci-archive:/tmp/iop-advisor-engine.tar \
-containers-storage:registry.redhat.io/satellite/iop-advisor-engine-rhel9:{ProjectVersion}
+# [{RepoRHEL9BaseOS}]
```

Comment:
```suggestion
[{RepoRHEL9BaseOS}]
```

You can have comments in `.repo` files, but each repo needs an ID. -> You should drop the leading `# ` in line 76, 82, and 89.

Example:

````
$ podman run --rm -it registry.access.redhat.com/ubi10/ubi:10.0 cat /etc/yum.repos.d/ubi.repo | head -n 6
[ubi-10-for-$basearch-baseos-rpms]
name = Red Hat Universal Base Image 10 (RPMs) - BaseOS
baseurl = https://cdn-ubi.redhat.com/content/public/ubi/dist/ubi10/10/$basearch/baseos/os
enabled = 1
gpgkey = file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
gpgcheck = 1
````

FYI: If you'd drop lines starting with `[`, `dnf makecache` will return an error because it cannot read the file anymore.